_layouts/home.html

```diff
@@ -38,8 +38,14 @@
       }, 1250, 'easeInOutExpo');
       event.preventDefault();
     });
+    var x = window.matchMedia("(max-width: 768px)");
```

Comment:
Not your fault (I know!), but all this is a hedious design, totally against the SoC... Janitors material for the future...

Reply:
Hahaha, Separation of Concerns!